README.md

```diff
@@ -10,6 +10,7 @@
 - Download the latest release for your architecture from the [Github Releases](https://github.com/keboola/keboola-as-code/releases).
 - Extract archive, eg. `unzip kbc-vx.x.x-linux_arm64.zip`.
 - Make binary executable, eg. `chmod +x ./kbc`.
+- for macOS >v10.15 - binary as an macOS application is not yet signed, and you may encounter error saying `“kbc” cannot be opened because it is from an unidentified developer.`. In such case add the binary to the allowed applications by running command `spctl --add ./kbc`
```

Comment:
Malo by to zacinat asi velkym pismenom ako ostatne vety:
![image](https://user-images.githubusercontent.com/19371734/125794088-39bdaa79-c108-44a1-9416-3a0fc5c03a82.png)

+ mozno by to bolo dobre rozdelit na nejake pod-odrazky aby to bolo prehladnejsie (uz ako chces), napr:
```
- Fix permissions on MacOs > v10.15:
  - ...
  - ....
```